tools/draw-importances.py

```diff
@@ -75,21 +81,25 @@ def blockImportance(input, verbose, path, raw):
         for bin in bin_list:
             with open(bin, 'rb') as f:
                 contents_batch = (f.read())
-                rows, cols, = struct.unpack('qq', contents_batch[:16])
-                imps = np.frombuffer(contents_batch[16:],
+                rows, cols, frame_type = struct.unpack('qqq', contents_batch[:24])
+                imps = np.frombuffer(contents_batch[24:],
                                     dtype=np.float32).reshape((rows, cols))
                 imps_list.append(imps)
                 rows_list.append(rows)
                 col_list.append(cols)
+                frame_type_list.append(frame_type)
+                mean_list.append(np.mean(imps))
 
         if raw:
             click.secho("The full imps data after processing:", fg="red")
             click.echo(imps_list)
 
     if verbose and path != None:
+        click.echo("Frame Type List: "+ str(frame_type_list))
```

Comment:
This output of data is inconsistent with the lines below.
```suggestion
        click.secho("Frame Type List: "+ str(frame_type_list))
```